.github/workflows/pull-request-conditionals.yaml

```diff
@@ -178,7 +165,7 @@ jobs:
   # Shim required checks for backport PRs to release branches (i.e., non-release-please PRs)
   run-package-test-shim:
     needs: check-paths
-    if: ${{ needs.check-paths.outputs.run_full_tests != 'true' && startsWith(github.base_ref, 'release/') }}
+    if: ${{ needs.check-paths.outputs.run_full_tests != 'true' && startsWith(github.base_ref, 'release/') && needs.check-paths.outputs.combined != '[]' }}
```

Comment:
I think this should run regardless of `combined` output (still conditional on the other checks).